### PR TITLE
Fixes #8660: Adapt documetation url of each method to us rudder documentation if editor is used with Rudder

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/8660

Replacing previous PR: https://github.com/Normation/ncf/pull/402

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
Commit modified

File: api/ncf_api_flask_app/views.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 sys.path[0:0] = [NCF_TOOLS_DIR]
 
 default_path = ""
+use_rudder_auth = True
```
Comment (fanf) [inline api/ncf_api_flask_app/views.py]: Not sure about that value. Why Rudder by default in ncf? 
On the other hand, the auth was already set to Rudder by default...

Comment (VinceMacBuche) [inline api/ncf_api_flask_app/views.py]: We already set Rudder as default auth method, so it easier to change it no (juste chnage the flag) 

Best way should be to have this value in a config file, but that would imply changing in the packaging too

**Comment by VinceMacBuche** [comment]
Commit modified

**Pull request merged**